### ISSUE TITLE
Fitzjohn's Avenue not showing on Zoopla

### ISSUE THREAD

**Issue by therealmyth** [ActivateMedia/heathgate#34]
Hi Viv, 

I'm looking at it and will come back to you.

Fred

**Comment by therealmyth** [comment]
Hi Viv, 

After investigation, the issue was with the property type (Conversion Flat) not being supported. I changed the export script to default to Flat when a property type is not recognized. The live script already has that functionality. I ran the export manually and the property should appear on the portals shortly. 

Fred

**Comment by therealmyth** [comment]
Hi Viv, 

I was away this weekend and yesterday, I will look into it today.

Fred

**Comment by therealmyth** [comment]
Hi Viv, 

The sales one seems to be fine:  https://www.zoopla.co.uk/for-sale/details/44060691

I had a look at the rental, and the address for the property was "25 Fitzjohn's Ave, London NW3, 5NB", I changed to "25 Fitzjohn's Ave, London, NW3 5NB" and relaunch the export manually,  I think the comma in the middle of the postcode was the issue for that one. The script takes a bit to run, will monitor and get back to you to confirm it's up. 

Fred

**Comment by therealmyth** [comment]
Hi Viv, it looks like it fixed the issue: https://www.zoopla.co.uk/to-rent/details/44442851

It may be a good idea in the near future to change the back-end to let you enter each address components separately, with format validation on each to avoid these errors. 

Most of the time the error is due to an incomplete postcode, which is reported straight away on the live feed but takes a while on the bulk feed to figure out. An even on the live feed, if google returns just part of the post code, the script will try to figure out the exact post code base on the latitude/longitude of the property but this isn't always right. 

The other issue we get often is with the property type, which is a manual mapping from what you type in the back-end, and a numeric value provided by the feed spec. I've added a work around for it by defaulting to flat when the type is not known, but this can lead to some misleading info for example, if it's a house, and the property type is misspelled, it will show as a flat on the portals. We could look into fixing the list to avoid that in the future. 

Lastly, I've chased Zoopla today to get an update and see when we can go live with the real time feed. They were supposed to come back to me on Thursday but I haven't heard from them. Hopefully we should hear from them tomorrow. 

Fred
